server/src/apub/mod.rs

```diff
@@ -142,6 +155,34 @@ fn is_apub_id_valid(apub_id: &Url) -> bool {
 pub trait ToApub {
   type Response;
   fn to_apub(&self, conn: &PgConnection) -> Result<Self::Response, Error>;
+  fn to_tombstone(&self) -> Result<Tombstone, Error>;
+}
+
+fn create_tombstone(
+  deleted: bool,
+  object_id: &str,
+  updated: Option<NaiveDateTime>,
+  former_type: String,
```

Comment:
Cleaner to pass in a type here (if its not too complicated).

Reply:
It is unfortunately, because the page, note are `object::kind`, while the community one is `actor::kind`